# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1316996126 14400
#      Sun Sep 25 20:15:26 2011 -0400
# Node ID a6becd1b846e49d3cedfd1b9865ec2ad99918994
# Parent  359153d26cc13f53ae387c69f10bfc3796ff33ea
eliminate DO_SIMPLE_FOR_LOOP_ONCE macro definition too

* pt-eval.cc (DO_SIMPLE_FOR_LOOP_ONCE): Delete macro.

diff --git a/src/pt-eval.cc b/src/pt-eval.cc
--- a/src/pt-eval.cc
+++ b/src/pt-eval.cc
@@ -276,28 +276,16 @@ quit_loop_now (void)
                || tree_continue_command::continuing);
 
   if (tree_break_command::breaking)
     tree_break_command::breaking--;
 
   return quit;
 }
 
-#define DO_SIMPLE_FOR_LOOP_ONCE(VAL) \
-  do \
-    { \
-      ult.assign (octave_value::op_asn_eq, VAL); \
- \
-      if (! error_state && loop_body) \
-        loop_body->accept (*this); \
- \
-      quit = quit_loop_now (); \
-    } \
-  while (0)
-
 void
 tree_evaluator::visit_simple_for_command (tree_simple_for_command& cmd)
 {
   if (error_state)
     return;
 
   if (debug_mode)
     do_breakpoint (cmd.is_breakpoint ());
